src/main/java/gregtech/api/items/toolitem/ToolDefinitionBuilder.java

```diff
@@ -312,6 +320,11 @@ public Object2IntMap<Enchantment> getDefaultEnchantments(ItemStack stack) {
                 return this.defaultEnchantments;
             }
 
+            @Override
+            public Object2IntMap<Enchantment> getDefaultEnchantmentLevelIncrease(ItemStack stack) {
+                return this.defaultEnchantmentLevelIncreases;
```

Comment:
To prevent issues similar to this, we should guard collections like `IGTToolDefinitions#getDefaultEnchantments` using unmodifiable wrapper like `Collections#unmodifiableList`, `Object2IntMaps#unmodifiable` etc.